src/test/RwaSpell.t.sol

```diff
@@ -804,6 +804,20 @@ contract DssSpellTest is DSTest, DSMath {
         assertTrue(oracle.good("RWA001-A"));
     }
 
+    function testSpellIsCast_RWA001_INTEGRATION_CURE() public {
```

Comment:
You need to redeploy everything in order to take the changes you are doing in `cure`, then this test will fail.

Reply:
Otherwise it is calling a `oracle` that doesn't implement the new constraints in `cure`.

Reply:
Same comment here as the other PR too: we need to merge upstream back into this and update `vote()` and `voteTemp()`.